test/test-cases/scale/saic/test_sai_vnet_route_unidirectional.py

```diff
@@ -153,28 +154,59 @@ def test_vnet_route_unidirectional_create(self, dpu):
+        # No Route. send packets from a port, reflects the packets on the same port
+        SRC_VM_IP = "10.1.1.10"
+        OUTER_SMAC = "00:00:05:06:06:06"
+        OUR_MAC = "00:00:02:03:04:05"
+
+        VIP = TEST_VNET_ROUTE_UNIDIRECTIONAL_CONFIG['DASH_VIP']['vpe']['IPV4']
+        VNET_VNI = TEST_VNET_ROUTE_UNIDIRECTIONAL_CONFIG['DASH_VNET']['vnet']['VNI']
+        DIR_LOOKUP_VNI = TEST_VNET_ROUTE_UNIDIRECTIONAL_CONFIG['DASH_DIRECTION_LOOKUP']['dle']['VNI']
+        SRC_VM_PA_IP = TEST_VNET_ROUTE_UNIDIRECTIONAL_CONFIG['DASH_ENI']['eni']['VM_UNDERLAY_DIP']
+        ENI_MAC = TEST_VNET_ROUTE_UNIDIRECTIONAL_CONFIG['DASH_ENI_ETHER_ADDRESS_MAP']['eam']['MAC']
+        DST_CA_IP = TEST_VNET_ROUTE_UNIDIRECTIONAL_CONFIG['DASH_OUTBOUND_CA_TO_PA']['ocpe']['DIP']
+        DST_PA_IP = TEST_VNET_ROUTE_UNIDIRECTIONAL_CONFIG['DASH_OUTBOUND_CA_TO_PA']['ocpe']["UNDERLAY_DIP"]
+        DST_CA_MAC = TEST_VNET_ROUTE_UNIDIRECTIONAL_CONFIG['DASH_OUTBOUND_CA_TO_PA']['ocpe']["OVERLAY_DMAC"]
+        
+        # Send packet one
+        inner_pkt = simple_udp_packet(eth_dst = "02:02:02:02:02:02",
+                                      eth_src = ENI_MAC,
+                                      ip_dst  = DST_CA_IP,
+                                      ip_src  = SRC_VM_IP)
+        vxlan_pkt = simple_vxlan_packet(eth_dst         = OUR_MAC,
+                                        eth_src         = OUTER_SMAC,
+                                        ip_dst          = VIP,
+                                        ip_src          = SRC_VM_PA_IP,
+                                        udp_sport       = 11638,
+                                        with_udp_chksum = False,
+                                        vxlan_vni       = DIR_LOOKUP_VNI,
+                                        inner_frame     = inner_pkt)
+
+        # Expected received packet one
+        inner_exp_pkt = simple_udp_packet(eth_dst = DST_CA_MAC,
+                                          eth_src = ENI_MAC,
+                                          ip_dst  = DST_CA_IP,
+                                          ip_src  = SRC_VM_IP)
+        vxlan_exp_pkt = simple_vxlan_packet(eth_dst         = "00:00:00:00:00:00",
+                                            eth_src         = "00:00:00:00:00:00",
+                                            ip_dst          = DST_PA_IP,
+                                            ip_src          = VIP,
+                                            udp_sport       = 0,
+                                            with_udp_chksum = False,
+                                            vxlan_vni       = VNET_VNI,
+                                            vxlan_flags     = 0,
+                                            inner_frame     = inner_exp_pkt)
+        vxlan_exp_pkt['IP'].chksum = 0
+
+        # Send packets from port 0
+        send_packet(dataplane, 0, vxlan_pkt, 10)
+        time.sleep(0.5)
+
+        # Verify packets from port 1
```

Comment:
Did you mean to write port 0?

Reply:
Not intentional. the packet can be sent from any of the two ports.

Reply:
I wasn't concerned about which port you wrote to. Your comment says
>        # Verify packets from port 1
But the code is using port 0. I think they should agree.

Reply:
Oh yea, my bad updated the comment and pushed the file. Thanks for the catch